cli/src/value/name_id.rs

```diff
@@ -0,0 +1,24 @@
+use std::{str::FromStr, num::ParseIntError};
+
+#[derive(Clone, Debug)]
+pub struct NameIdPair {
```

Comment:
@jules module must be under cli::value::name_id_pair 
and re-export in cli::value